### ISSUE TITLE
train loss No change,No change in predicted value

### ISSUE THREAD

**Issue by da-suda** [lululxvi/deepxde#733]
pde：
![pde](https://user-images.githubusercontent.com/72438519/173242596-7f36dbf9-e5a1-4eb8-b2e5-74679ed88829.png)
I use part of the known results as boundary conditions, ex is another variation parameter, using the method in Lorenz attractor I write him, after training, the training loss e1 is decreasing, but e2 does not change, and the predicted value does not change. The code is as follows
<img width="529" alt="5" src="https://user-images.githubusercontent.com/72438519/173243024-a5e55416-f4ae-47c5-9c1b-6b8969517fe6.PNG">
<img width="272" alt="2" src="https://user-images.githubusercontent.com/72438519/173242868-108893c1-711a-4a32-ab00-1fd907aaf2fd.PNG">
<img width="273" alt="3" src="https://user-images.githubusercontent.com/72438519/173242876-ac4c6914-cf25-4d05-9180-c9ab8f17d22c.PNG">
<img width="280" alt="4" src="https://user-images.githubusercontent.com/72438519/173242883-99169266-ee9e-4b4c-8c82-745713b1bc18.PNG">
<img width="276" alt="6" src="https://user-images.githubusercontent.com/72438519/173243042-4b3d8674-7e7b-427c-bc2f-72a637e39960.PNG">
<img width="344" alt="7" src="https://user-images.githubusercontent.com/72438519/173243045-2b2c429d-c831-4ee4-af26-0d3b96d5fc7f.PNG">
<img width="333" alt="8" src="https://user-images.githubusercontent.com/72438519/173243049-a7bb4e06-8a4a-465d-995b-24524e835625.PNG">

code：
def load_train_data(num):
    t_star = scipy.io.loadmat('t.mat')['dt'].T[10000:14000] #18001 x 1
    z_star = scipy.io.loadmat('z.mat')['dz'].T[600:630]  #641 x 1
    p_star = scipy.io.loadmat('Ps.mat')['Ps'].T[600:630, 10000:14000] #641 x 18001
    v_star = scipy.io.loadmat('V.mat')['V'].T[600:630, 10000:14000]#641 x 18001
    pf_star = scipy.io.loadmat('pf.mat')['Pf'][10000:14000]#18001 x 1

    T = t_star.shape[0]
    N = z_star.shape[0]

    ZZ = np.tile(z_star,(1, T)) #641 x 18001
    TT = np.tile(t_star, (1, N)).T #641 x 18001
    PFF = np.tile(pf_star, (1, N)).T #641 x 18001
    PP = p_star
    VV = v_star

    z = ZZ.flatten()[:,None] #641*18001 x 1
    t = TT.flatten()[:,None] #641*18001 x 1
    p = PP.flatten()[:,None] #641*18001 x 1
    v = VV.flatten()[:,None] #641*18001 x 1
    pf = PFF.flatten()[:,None] #641*18001 x 1

    idx = np.random.choice(N*T, num, replace=False)
    z_train = z[idx,:]
    t_train = t[idx,:]
    p_train = p[idx,:]
    v_train = v[idx,:]
    pf_train = pf[idx,:]

    return [z_train, t_train, p_train, v_train, pf_train]

z_lower = 600
z_upper = 630
t_lower = 10
t_upper = 14
space_domain = dde.geometry.Interval(z_lower, z_upper)
time_domain = dde.geometry.TimeDomain(t_lower, t_upper)
geomtime = dde.geometry.GeometryXTime(space_domain, time_domain)

[ob_z, ob_t, ob_p, ob_v, ob_pf] = load_train_data(num=10000)

plt.scatter(ob_t,ob_p)
plt.xlabel("time")
plt.ylabel("p ")
plt.show()
plt.scatter(ob_t,ob_v)
plt.xlabel("time")
plt.ylabel("v")
plt.show()

ob_zt = np.hstack((ob_z,ob_t))
observe_p = dde.icbc.PointSetBC(ob_zt, ob_p, component=0)
observe_v = dde.icbc.PointSetBC(ob_zt, ob_v, component=1)

t_fun = scipy.io.loadmat('t.mat')['dt'].T[10000:14000] #18001 x 1
pf_fun = scipy.io.loadmat('pf.mat')['Pf'][10000:14000]#18001 x 1
plt.plot(t_fun, pf_fun)
plt.xlabel("time")
plt.ylabel("pf")
plt.show()

def ex_fun(x):
    spline = scipy.interpolate.Rbf(t_fun, pf_fun, function="thin_plate", smooth=0, epsilon=0)
    return spline(x[:, 1:])

def pde(x, y, ex):

    C = 919.4902
    rou = 1294

    p = y[:, 0:1]
    v = y[:, 1:2]

    p_t = dde.grad.jacobian(y, x, i=0, j=1)
    v_t = dde.grad.jacobian(y, x, i=1, j=1)

    p_z = dde.grad.jacobian(y, x, i=0, j=0)
    v_z = dde.grad.jacobian(y, x, i=1, j=0)

    e1 = v*p_z + p_t + rou*(C**2)*v_z
    e2 = p_z + rou*v_t + rou*v*v_z + ex

    return [e1,e2]

data = dde.data.TimePDE(
    geomtime,
    pde,
    [observe_p,observe_v],
    num_domain=15000,
    num_boundary=20,
    num_initial=10,
    anchors=ob_zt,
    auxiliary_var_function=ex_fun
    # train_distribution="pseudo"
)

net = dde.nn.FNN([2] + [100]*4 + [2], "tanh", "Glorot normal")
model = dde.Model(data, net)

model.compile("adam", lr=1e-3, loss="MSE")
model.train(epochs=15000)

model.compile("L-BFGS")
losshistory, train_state = model.train()
dde.saveplot(losshistory, train_state, issave=True, isplot=True)

**Comment by lululxvi** [comment]
Your loss is 5e12. See FAQ Q: I failed to train the network or get the right solution, e.g., large training loss, unbalanced losses.

**Comment by da-suda** [comment]
> Your loss is 5e12. See FAQ Q: I failed to train the network or get the right solution, e.g., large training loss, unbalanced losses.
Thank you for your reply
1、I narrowed the time domain 0-5s
2、Added network layer
3、Modified the weights
But my loss is still big  2e11
and e2 No change in the loss of this pde
Could it be my pde problem?This method is not suitable for solving my pde？

**Comment by lululxvi** [comment]
There are more things you can try, see more at FAQ.
